src/graphql/query/customSecondaryMenuQuery.graphql

```diff
@@ -0,0 +1,3 @@
+query {
+	usingTouch @client
+}
```

Comment:
Please rename this file to something like "usingTouchClient.graphql" as it's so general we can use it elsewhere.

Reply:
Agreed, added it to a `shared` folder and named it `usingTouchClient.graphql`